kasa/discover.py

```diff
@@ -264,6 +264,7 @@ def __init__(
         self.target = target
         self.target_1 = (target, self.discovery_port)
         self.target_2 = (target, Discover.DISCOVERY_PORT_2)
+        self.target_3 = (target, Discover.DISCOVERY_PORT_3)
```

Comment:
Wasn't this added already in an earlier PR? If so, please rebase on top of the current master branch.